specification/marketplacecatalog/resource-manager/Search/stable/2022-02-02/search.json

```diff
@@ -0,0 +1,785 @@
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "azureCategories",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "azurePortalCategories",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "array",
+            "name": "facets",
+            "in": "query",
+            "collectionFormat": "multi",
+            "x-nullable": true,
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "string",
+            "name": "x-ms-app",
+            "x-originalName": "callingApp",
+            "in": "header",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "x-originalName": "apiVersion",
+            "in": "query",
+            "required": true,
+            "default": "2022-02-02",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "Returns a list of azure private store marketplace catalog offers and total count and facets",
+            "schema": {
+              "$ref": "#/definitions/SearchResponseDto"
+            }
+          },
+          "400": {
+            "x-nullable": false,
+            "description": "In case request malformed",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "#/definitions/SearchErrorResponseDto"
+            }
+          },
+          "500": {
+            "x-nullable": false,
+            "description": "In case internal error",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "#/definitions/SearchErrorResponseDto"
+            }
+          },
+          "default": {
+            "description": "In case internal error",
+            "schema": {
+              "$ref": "#/definitions/SearchErrorResponseDto"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all of the available Microsoft.Marketplace REST API operations.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
```

Comment:
>#/definitions/OperationListResult [](http://example.com/codeflow?start=23&length=33)

Use types from v3 common types unless you have a compelling reason not to.

Reply:
@mentat9 can you please share an example of such usage?